src/emsdevice.cpp

```diff
@@ -768,7 +768,7 @@ void EMSdevice::generate_values_web(JsonObject & output) {
                     if (divider > 0) {
                         obj["s"] = Helpers::render_value(s, (float)1 / divider, 1);
                     } else if (divider < 0) {
-                        obj["s"] = Helpers::render_value(s, (-1) * divider, 0);
+                        obj["s"] = Helpers::render_value(s, (float)(-1) * divider, 0);
```

Comment:
what is the reason behind adding this?

Reply:
It's interesting because this was causing compile issues for me even without any modification. (src/emsdevice.cpp:771:78: error: call of overloaded 'render_value(char [10], int, int)' is ambiguous) I'm using the 5.1.1 version of the espriff 32 lib.